smartsim/entity/application.py

```diff
@@ -247,7 +247,7 @@ def attached_files_table(self) -> str:
         :returns: String version of table
         """
         if not self.files:
-            return "No file attached to this application."
+            raise ValueError("No file attached to this application.")
```

Comment:
We allow users to specify EntityFiles or None, so this will catch and throw even though we do not require files, I agree to remove this check and just return an empty list!

Reply:
changed